frontend/ui/widget/dictquicklookup.lua

```diff
@@ -339,7 +339,7 @@ function DictQuickLookup:update()
                 },
                 {
                     text = self:getHighlightText(),
-                    enabled = true,
+                    enabled = self.highlight ~= nil,
```

Comment:
I knew it'd be simple but I didn't think it'd be that simple or I would've just done that instead of writing an issue. :-P

Reply:
:)